Adjust the number of yescrypt rounds.

--- /etc/pam.d/common-password	2023-07-25 13:20:42.674294833 +0200
+++ /etc/pam.d/common-password	2023-07-25 13:25:17.926204195 +0200
@@ -22,7 +22,7 @@
 # pam-auth-update(8) for details.
 
 # here are the per-package modules (the "Primary" block)
-password	[success=1 default=ignore]	pam_unix.so obscure yescrypt
+password	[success=1 default=ignore]	pam_unix.so obscure yescrypt rounds=@ROUNDS@
 # here's the fallback if no module succeeds
 password	requisite			pam_deny.so
 # prime the stack with a positive return value if there isn't one already;
